### PR TITLE
Add .NET Monitor to set versions script.

### PR DESCRIPTION
**Pull request by jander-msft**
cc @dotnet/dotnet-monitor

### CODE REVIEW

**Review by mthalman — COMMENTED** [review]
This requires providing the `--channel-name` option too, right?

Can you also update the release issue template accordingly:
https://github.com/dotnet/dotnet-docker/blob/a0d8965e08cba729071556957bf8830c82cb0be0/.github/ISSUE_TEMPLATE/releases/dotnet-monitor-release.md?plain=1#L14-L15

**Comment by jander-msft** [comment]
> This requires providing the `--channel-name` option too, right?

I looked through the update-dependencies code and it looks like we are not using it anymore because our package and checksum files are now under build version folders. So it's not necessary anymore. I was thinking of removing the notion in a change after the release went out the door and not adding more to its "debt", if that makes sense.

> Can you also update the release issue template accordingly:
> 
> https://github.com/dotnet/dotnet-docker/blob/a0d8965e08cba729071556957bf8830c82cb0be0/.github/ISSUE_TEMPLATE/releases/dotnet-monitor-release.md?plain=1#L14-L15

Absolutely.

File: .github/ISSUE_TEMPLATE/releases/dotnet-monitor-release.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
       - [ ] Check for additional changes by diffing the main and nightly branches
       - [ ] &lt;add link to PR/commit&gt;
 1. - [ ] Run `update-dependencies` tool to update all the necessary files to reflect the specified .NET versions (run this command for each version being released):
-      - [ ] `dotnet run --project .\eng\update-dependencies\update-dependencies.csproj <major/minor version> --product-version monitor=<version> --channel-name <channel>`
+      - [ ] `./eng/Set-DotnetVersions.ps1 -ProductVersion <major/minor version> -MonitorVersion monitor=<version>`
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/releases/dotnet-monitor-release.md:15]:
```suggestion
      - [ ] `./eng/Set-DotnetVersions.ps1 -ProductVersion <major/minor version> -MonitorVersion <version>`
```

**Review by mthalman — APPROVED** [review]
Just one tweak to the issue template needed.

**Pull request merged**